pom.xml

```diff
@@ -36,7 +36,7 @@
             <url>http://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
-
+    
```

Comment:
For future reference, it is nice to check your diffs when submitting PRs or PR revisions to make sure they are minimal. If you want to revert everything from a given file,

    git checkout master -- pom.xml

while reverting individual hunks is a bit more work.